src/main/java/com/stripe/model/Payout.java

```diff
@@ -123,6 +123,13 @@ public void setDestinationObject(ExternalAccount c) {
 		this.destination = new ExpandableField<ExternalAccount>(c.getId(), c);
 	}
 
+	public Boolean isDestinationDeleted() {
+		if (this.destination == null) {
+			return false;
```

Comment:
Note that this means that we'd return `false` when the object is not expanded, even though in that case we don't know if the resource is deleted or not. Maybe it'd be better to raise an exception, e.g. "you should not call this method if you did not expand the attribute"?